### ISSUE TITLE
failed to build plugin for UE5 preview 

### ISSUE THREAD

**Issue by ando9527** [fpwong/BlueprintAssistWiki#77]
Build with 4.27 plugin, I got the following error.

```cpp
  BlueprintAssistCreateAssetMenu.cpp(126): [C2440] 'initializing': cannot convert from 'FContentBrowserItemPath' to 'FString'
  BlueprintAssistCreateAssetMenu.cpp(126): [C2440] No constructor could take the source type, or constructor overload resolution was ambiguous
  BlueprintAssistInputProcessor.cpp(1938): [C2668] 'TUnrealPlatformMathSSEBase<Base>::RoundToInt': ambiguous call to overloaded function
  UnrealPlatformMathSSE.h(281): [C2668] could be 'int32 TUnrealPlatformMathSSEBase<Base>::RoundToInt(double)'
  UnrealPlatformMathSSE.h(280): [C2668] or       'int32 TUnrealPlatformMathSSEBase<Base>::RoundToInt(float)'
  BlueprintAssistInputProcessor.cpp(1938): [C2668] while trying to match the argument list '(const int)'
  BlueprintAssistInputProcessor.cpp(1948): [C2668] 'TUnrealPlatformMathSSEBase<Base>::RoundToInt': ambiguous call to overloaded function
  UnrealPlatformMathSSE.h(281): [C2668] could be 'int32 TUnrealPlatformMathSSEBase<Base>::RoundToInt(double)'
  UnrealPlatformMathSSE.h(280): [C2668] or       'int32 TUnrealPlatformMathSSEBase<Base>::RoundToInt(float)'
  BlueprintAssistInputProcessor.cpp(1948): [C2668] while trying to match the argument list '(const int)'
  Microsoft.MakeFile.targets(45, 5): [MSB3073] The command ""D:\Program Files\Epic Games\UE_5.0\Engine\Build\BatchFiles\Build.bat" GamejamEditor Win64 Development -Project="D:\perforce\gj5\Gamejam\Gamejam.uproject" -WaitMutex -FromMsBuild" exited with code 6.```

**Comment by fpwong** [comment]
Hi, you can download a built version of the plugin for 5.0 here: https://drive.google.com/file/d/1J6ShF2x59YmWFBhYtDoDE3SZRrCqS09e/view

I am in the process of fixing the deprecated functions, so if you do build it again you will see a bunch of warning messages.

**Comment by fpwong** [comment]
Build issues for UE5 should be fixed as of version 3.2.14.
